From f1417ab403aefb0446f59018a8360afc3895b51b Mon Sep 17 00:00:00 2001
From: Nikolaus Rath <Nikolaus@rath.org>
Date: Mon, 1 Feb 2016 08:40:44 -0800
Subject: =?UTF-8?q?From=204baa45092b0517cb905c0993610885f52a09c735=20Mon?=
 =?UTF-8?q?=20Sep=2017=2000:00:00=202001=0AFrom:=20Nikolaus=20Rath=20<Niko?=
 =?UTF-8?q?laus@rath.org>=0ADate:=20Thu,=208=20Oct=202015=2012:24:35=20-07?=
 =?UTF-8?q?00=0ASubject:=20Force=20use=20of=20libattr1-dev=20API?=

Forwarded: not-needed
Last-Update: 2015-08-21
 Compilation under GNU/kFreeBSD fails because none of upstream's
 preprocessor macros matches this combination. It is also not clear
 how to properly test for the extended attribute API without
 introducing something heavyweight like autoconf. In Debian, however,
 the libattr1-dev API is available for all supported architectures,
 so it is safe to just hardcode the use of this API. This is what
 this patch does.
 .

Patch-Name: force_xattr_api.diff
---
 src/llfuse.h | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/src/llfuse.h b/src/llfuse.h
index 159e..1606 100644
--- a/src/llfuse.h
+++ b/src/llfuse.h
@@ -12,18 +12,7 @@ the terms of the GNU LGPL.
 #define PLATFORM_BSD 2
 #define PLATFORM_DARWIN 3
 
-#ifdef __gnu_linux__
 #define PLATFORM PLATFORM_LINUX
-#elif __FreeBSD__
-#define PLATFORM PLATFORM_BSD
-#elif __NetBSD__
-#define PLATFORM PLATFORM_BSD
-#elif __APPLE__ && __MACH__
-#define PLATFORM PLATFORM_DARWIN
-#include "darwin_compat.h"
-#else
-#error "Unable to determine system (Linux/FreeBSD/NetBSD/Darwin)"
-#endif
 
 #include <fuse.h>
 
